### PR TITLE
Feature/login style new

### PR DESCRIPTION
**Pull request by paul1124**

### CODE REVIEW

**Review by nroode — CHANGES_REQUESTED** [review]
Hi Paul,

A couple things:
- On the login/sign up, the spacing is off.  There should be an equal padding around the entire window.  Right now looks like there is extra margin on the 'login' that isn't needed.  The padding on the bottom is also being cut off. 
- I would also decrease the size of the app title - and make sure there's also an equal amount of padding around it on all sides.
- On the login page - I would not have the full 'Already have an account? Sign in here' as a button.  I would bold the sign in here part like the design, and have only this be a button.  Let's also remove the strange green background hover that is applied.
- Doesn't look like the header login/signup buttons do anything - could you remove? 
- Could you increase the width of the login/signup block? Also make the Title in all caps like the design.
- Submit button on login page is bleeding/overlapping with the padding

- Again on the login page, could you change the text from 'Don't have an account? Sign up for FREE' to 'Don't have an account? Sign up here' and again make only the sign up here part the button.  
- On the app list page, when you hover over the App Listing, there's a strange green 'button' that shows up.  I had previously fixed this but it looks like it was overridden so you'll need to address this again.

File: client/package.json [hunk 1]
```diff
@@ -2,8 +2,13 @@
 	"name": "client",
 	"version": "1.0.0",
 	"dependencies": {
+		"@fortawesome/fontawesome-svg-core": "^1.2.17",
+		"@fortawesome/free-solid-svg-icons": "^5.8.1",
+		"@fortawesome/react-fontawesome": "^0.1.4",
```
Comment (terrancexin) [inline client/package.json:7]: what are the needs for this?

File: client/package.json [hunk 2]
```diff
@@ -2,8 +2,13 @@
...
 		"axios": "^0.18.0",
+		"i": "^0.3.6",
```
Comment (terrancexin) [inline client/package.json:9]: what is this?

File: client/src/Components/Header/index.jsx [hunk 3]
```diff
@@ -2,11 +2,14 @@ import React from 'react';
...
+export default function Header({ setAuth }) {
 	return (
 		<header>
-			<h1 className="header">Job Tracker</h1>
-			<h1>{currentUser}</h1>
+			<h1>Job Tracker</h1>
 		</header>
 	);
+
+	function handleClick() {
```
Comment (terrancexin) [inline client/src/Components/Header/index.jsx]: handleClick is not being used anywhere

File: client/package.json (hunk 1 above)
Comment (paul1124) [reply client/package.json:7]: For the checkmark icons in the login page.

File: client/package.json (hunk 2 above)
Comment (paul1124) [reply client/package.json:9]: This I don't know; I don't remember adding it

Comment (terrancexin) [reply client/package.json:9]: any lines of code that you're not using and newly added by this PR should be removed

File: client/package.json [hunk 4]
```diff
@@ -2,8 +2,13 @@
...
 		"node-sass": "^4.11.0",
+		"npm": "^6.9.0",
```
Comment (terrancexin) [inline client/package.json:11]: remove

File: client/src/Components/Header/index.jsx [hunk 5]
```diff
@@ -2,11 +2,10 @@ import React from 'react';
 
 import './header.scss';
 
-export default function Header({ currentUser }) {
+export default function Header({ setAuth }) {
```
Comment (terrancexin) [inline client/src/Components/Header/index.jsx:5]: remove setAuth

File: client/package.json (hunk 1 above)
Comment (nroode) [reply client/package.json:7]: I just used the cdn link to keep it simple elsewhere in the app so we wouldn't need to use both.

**Pull request merged**